src/Raven.Server/Utils/Cli/RavenCli.cs

```diff
@@ -353,10 +353,8 @@ private static bool CommandStats(List<string> args, RavenCli cli)
 
             LoggingSource.Instance.DisableConsoleLogging();
             var prevLogMode = LoggingSource.Instance.LogMode;
-            SetupLogMode(LogMode.None, cli._server.Configuration.Logs);
-            Program.WriteServerStatsAndWaitForEsc(cli._server);
             SetupLogMode(prevLogMode, cli._server.Configuration.Logs);
```

Comment:
why we want to setup prev log mode before doing any of the operations? we didn't change it after all?